engine/lib/users.php

```diff
@@ -764,20 +764,32 @@ function users_pagesetup() {
 
 	// topbar
 	if ($viewer) {
+
+		elgg_register_menu_item('topbar', array(
+			'name' => 'account',
+			'text' => elgg_echo('account'),
+			'href' => "#",
+			'priority' => 100,
+			'section' => 'alt',
+			'link_class' => 'elgg-topbar-dropdown',
+		));
+		
 		elgg_register_menu_item('topbar', array(
 			'name' => 'usersettings',
+			'parent_name' => 'account',
 			'href' => "settings/user/{$viewer->username}",
-			'text' => elgg_view_icon('settings') . elgg_echo('settings'),
-			'priority' => 500,
+			'text' => elgg_echo('settings'),
+			'priority' => 103,
```

Comment:
can we have a bit more room in priorities? it is pretty hard to put a menu item between them in this way... just make them 150 and 200 or something